README.md

```diff
@@ -489,6 +489,31 @@ $ git push heroku master
 If the app is deployed successfully, congratulations! You've finished setting
 up, deploying to, and hosting your bot on Heroku!
 
+## Troubleshooting
+
+Sometimes mistakes happen and like with most programming related problem - the solution is reading logs and documentation. 
```

Comment:
The grammar and punctuation here in this paragraph isn't correct, so it makes it awkward to read; for example, the dash would be better suited as a comma. Can you try rephrasing?